### PR TITLE
Added functions to set and get the name of the logger

### PR DESCRIPTION
**Pull request by BrodyHiggerson**

### CODE REVIEW

File: smolog.cpp [hunk 1]
```diff
@@ -56,6 +56,23 @@ namespace smolog {
 		remove_sink(s.get());
 	}
 
+	void logger::set_name(const char* name)
+	{
+		if (_internal)
+		{
+			_internal->name = name;
```
Comment (martty) [inline smolog.cpp]: Please change brace style to match existing.

Comment (BrodyHiggerson) [reply smolog.cpp]: Done!

Comment (martty) [reply smolog.cpp]: Please change it some more :)

File: smolog.cpp [hunk 2]
```diff
@@ -56,6 +56,19 @@ namespace smolog {
...
 	}
 
+	void logger::set_name(const char* name) {
+		if (_internal) {
+			_internal->name = name;
+		}
+	}
+
+	const char* logger::get_name() const {
+		if (_internal) {
```
Comment (martty) [inline smolog.cpp]: _internal is never null, please remove the check

**Pull request merged**